### PR TITLE
Swizzled TOC component to include CTA inside, then set position to re…

### PR DESCRIPTION
**Pull request by breezyfasano**
…lative so it wouldnt overlap

## Description & motivation

This just fixed a small bug as part of this [Asana task](https://app.asana.com/0/1200099998847559/1201624915841764/f)!

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

**Review by runleonarun — APPROVED** [review]
LGTM! Thanks for making this change, Breezy

**Review by lfracalossiFA — COMMENTED** [review]
Hi Breezy -- I have one question about what looked like a typo to me. (It might also be some functionality I'm not understanding). If that's intentional, this is approved; if that's an accident, then please consider this a change request.

Thanks!
Lise

File: website/src/theme/BlogLayout/index.js [hunk 1]
```diff
@@ -132,7 +132,7 @@
        
        <div className="mobile-toc-section container">
          {toc && (
-           <div className="mobile-toc-container">
+           <div className="mobile-toc-,4container">
```
Comment (lfracalossiFA) [inline website/src/theme/BlogLayout/index.js]: Huh, thought I left a comment here already...

Is this class name (`mobile-toc-,4container`) intentional, or a typo?

Comment (breezyfasano) [reply website/src/theme/BlogLayout/index.js]: Woop, I think that's a typo! Great catch. I pushed a fix.

**Review by JKarlavige — APPROVED** [review]
looks good!

**Comment by lfracalossiFA** [comment]
Typo looks fixed to me! Ship it 🚢

**Pull request merged**